### ISSUE TITLE
Error with bootloader install with LVM

### ISSUE THREAD

**Issue by EricGallimore** [DonyorM/weresync#15]
I'm not entirely sure what is causing this error, but hopefully the traceback will be more useful to someone more familiar with the project.

```
~$ sudo weresync -C -l maggie-vg -E 1 -g 3 -B 2 /dev/sda /dev/sdb
Checking partition validity.
Checking partition validity.
WARNING - 2018-05-11 18:23:41,483 - weresync.interface - Partition count on two drives different. Invalid.
Partitions invalid!
Copying drive partition table.
WARNING - 2018-05-11 18:23:41,484 - weresync.interface - Drives are incompatible.
Beginning to copy files.
Copying partition 1
Copying partition 2
WARNING - 2018-05-11 18:23:42,465 - weresync.device - Error copying data for partition 3 from device /dev/sda to /dev/sdb.
Finished copying files.
Copying partition root

WARNING - 2018-05-11 18:26:27,530 - weresync.device - Error copying data for partition swap_1 from device maggie--vg to /dev/maggie-vg-copy.
Finished copying files.
Finished copying files.
Making bootable
WARNING - 2018-05-11 18:26:27,710 - weresync.device - Error copying fstab on LVM. Continuing anyway.
Traceback (most recent call last):
  File "/usr/local/bin/weresync", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.5/dist-packages/weresync/interface.py", line 510, in main
    bootloader=args.bootloader, )
  File "/usr/local/lib/python3.5/dist-packages/weresync/interface.py", line 334, in copy_drive
    boot_partition, efi_partition, boot_callback)
  File "/usr/local/lib/python3.5/dist-packages/weresync/device.py", line 1687, in make_bootable
    root_partition, efi_partition)
  File "/usr/local/lib/python3.5/dist-packages/weresync/plugins/weresync_uuid_copy.py", line 52, in install_bootloader
    plugins.translate_uuid(copier, boot_partition, "/", target_mnt)
  File "/usr/local/lib/python3.5/dist-packages/weresync/plugins/__init__.py", line 75, in translate_uuid
    uuid_dict = copier.get_uuid_dict()
  File "/usr/local/lib/python3.5/dist-packages/weresync/device.py", line 870, in get_uuid_dict
    source_vg = parse.parse("/dev/{0}", self.lvm_source.device)[0]
TypeError: 'NoneType' object is not subscriptable
```

The disk I'm trying to copy looks like this (the default layout created by the Ubuntu 16.04 Server installer):
```
Disk /dev/sda: 465.8 GiB, 500107862016 bytes, 976773168 sectors
Units: sectors of 1 * 512 = 512 bytes
Sector size (logical/physical): 512 bytes / 512 bytes
I/O size (minimum/optimal): 512 bytes / 33553920 bytes
Disklabel type: gpt
Disk identifier: DEBCE901-CC7C-4BEF-9F73-9E0DD685A046

Device       Start       End   Sectors   Size Type
/dev/sda1     2048   1050623   1048576   512M EFI System
/dev/sda2  1050624   2050047    999424   488M Linux filesystem
/dev/sda3  2050048 976771071 974721024 464.8G Linux LVM
```

Running Python 3.5.2 with weresync installed via pip on Ubuntu 16.04.3 (Linux 4.4.0-124-generic #148-Ubuntu SMP Wed May 2 13:00:18 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux)

Log file is attached.
[weresync.log](https://github.com/DonyorM/weresync/files/1997010/weresync.log)

**Comment by DonyorM** [comment]
I believe this is fixed with version 1.0.9. Let me know if you have any more issues.

**Comment by jacblo** [comment]
yeah it says that for me too and it isn't working
